python_modules/libraries/dagster-databricks/dagster_databricks/databricks_pyspark_step_launcher.py

```diff
@@ -130,10 +132,11 @@ def __init__(
         databricks_host,
         databricks_token,
         secrets_to_env_variables,
-        storage,
         staging_prefix,
         wait_for_logs,
         max_completion_wait_time_seconds,
+        env_variables={},
```

Comment:
ok looks like there's a small linting issue with this setup (my bad). If you change this to env_variables=None, storage=None, and then change the check.dict_param calls to check.opt_dict_param() calls, then everything should sort itself out

Reply:
Updated.